x/gov/client/cli/tx.go

```diff
@@ -56,7 +57,7 @@ func GetCmdSubmitProposal(cdc *codec.Codec) *cobra.Command {
 		Long: strings.TrimSpace(`
 Submit a proposal along with an initial deposit. Proposal title, description, type and deposit can be given directly or through a proposal JSON file. For example:
 
-$ <appcli> gov submit-proposal --proposal="path/to/proposal.json" --from mykey
+` + fmt.Sprintf("$ %v gov submit-proposal --proposal=\"path/to/proposal.json\" --from mykey", version.ClientName) + `
```

Comment:
Certainly not a big deal, but why using `Sprintf()` multiple times here instead of wrapping the whole text with a single `Sprintf()` call?

Reply:
You can't accomplish this